### PR TITLE
Labels documentation extension

### PR DESCRIPTION
**Pull request by vterentev**
## Changes
 Added Bridges, basic implementation of scam, hacker, closed. Added several templates for labels pages, that we are planning to add in close future.
## Checklist

- [ ] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [x]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [x]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/labels/bridge/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description: Bridge label
+---
+
+## Description
+
+Crypto bridges enable the exchange of cryptocurrencies across different blockchains, 
+allowing users to access assets that were previously inaccessible. 
+They increase liquidity and interoperability between blockchain networks. Examples include Wrapped Bitcoin and the Binance Smart Chain.
+
+`Bridge` label is a part of [DeFi](/labels/defi) label.
```
Comment (alexander-gr) [inline src/docs/labels/bridge/index.md:14]: Do we really need that line?

File: src/docs/labels/cold-wallet/index.md [hunk 2]
```diff
@@ -7,14 +7,15 @@ description: Cold wallet label
 
 ## Description
 
-Denotes a cold wallet of a centralized exchanges or other entity.
+Label `cold_wallet` refers to a type of wallet that is commonly used by centralized exchanges or other entities as a secure storage solution for cryptocurrency assets.
 
+This label is the culmination of our in-depth analysis of both publicly available data and our own internal research, ensuring that our identification process is as comprehensive and accurate as possible.
```
Comment (alexander-gr) [inline src/docs/labels/cold-wallet/index.md:12]: Are you sure about that line?

File: src/docs/labels/dsproxy/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (alexander-gr) [inline src/docs/labels/dsproxy/index.md:1]: there's no such a label. DSproxy is a value for proxy label

File: src/docs/labels/fund/index.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ description: Fund label
 
 The fund label is used to identify wallet addresses that belong to investment funds operating in the cryptocurrency space.
 
+This label is the culmination of our in-depth analysis of both publicly available data and our own internal research, ensuring that our identification process is as comprehensive and accurate as possible.
```
Comment (alexander-gr) [inline src/docs/labels/fund/index.md:12]: ?

File: src/docs/labels/index.md [hunk 5]
```diff
@@ -116,7 +116,10 @@ Below you can find labels which are attached to every group from above. Please n
 - [Contract](/labels/contract)
     - [ERC721](/labels/erc721)
     - [ERC1155](/labels/erc1155)
-- [Hot wallet](/labels/hot-wallet)
+    - [Proxy](labels/proxy)
+    - TBA: Factory
+        - TBA: DSProxy
```
Comment (alexander-gr) [inline src/docs/labels/index.md:121]: There's no dsproxy label

File: src/docs/labels/lending/index.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+date: 2023-04-18
+description: Lending Labels
+---
+
+## Description
+
+Decentralized finance (DeFi) lending projects enable individuals to lend and borrow digital assets without intermediaries. 
+These platforms offer transparency, potentially higher interest rates, but also come with risks such as smart contract vulnerabilities and market volatility. 
+
+Lending belongs to [defi](/labels/defi).
```
Comment (alexander-gr) [inline src/docs/labels/lending/index.md:13]: same

File: src/docs/labels/owner/index.md [hunk 7]
```diff
@@ -7,11 +7,11 @@ description: Owner label
 
 ## Description
 
-_Owner_ is a standalone label which denotes the person or company which controls a given address. Note that owner label is not a nickname. Owner is a broader term rather than nickname which is an owner's attribute (e.g. Vitalik Buterin might have several nicknames: 'Vitaliy', 'ETH_daddy', 'Jogn Galt', etc).
+`Owner` is a standalone label which denotes the person or company which controls a given address. Note that owner label is not a nickname. Owner is a broader term rather than nickname which is an owner's attribute (e.g. Vitalik Buterin might have several nicknames: 'Vitaliy', 'ETH_daddy', 'Jogn Galt', etc).
```
Comment (alexander-gr) [inline src/docs/labels/owner/index.md:10]: We dont use `that style` for label names on other pages

File: src/docs/labels/proxy/index.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+the proxy contract immutable, you can easily upgrade the underlying target contract without having to redeploy the 
+entire system. This is a significant benefit as it saves time, resources, and reduces the risk of downtime during the 
+upgrade process.
+## API Name
+
+`proxy`
+
+## Label Examples
+
+MakerDAO Proxy Contract [0x00002c65ba1e64d2fd8c6ffb7160fd6bde9e17dd]
```
Comment (alexander-gr) [inline src/docs/labels/proxy/index.md:25]: `MakerDAO Proxy Contract` -> `MakerDAO DSProxy Contract`

File: src/docs/labels/proxy/index.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+title: Proxy
+author: Vladimir
+date: 2023-04-25
+description: Proxy label
+---
+
+## Description
+
+A proxy contract is a special type of contract that serves as an intermediary between the user and the actual contract.
```
Comment (alexander-gr) [inline src/docs/labels/proxy/index.md:10]: Doesnt fir

File: src/docs/labels/staking/index.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+date: 2023-04-18
+description: Liquid Staking Label
+---
+
+## Description
+
+DeFi liquid staking projects create tokenized versions of staked assets, allowing users to earn rewards while maintaining liquidity. 
+These projects provide increased flexibility and higher rewards compared to traditional staking.
+
+Liquid staking group is a part of  [DeFi Group](/labels/defi)
```
Comment (alexander-gr) [inline src/docs/labels/staking/index.md:13]: same

File: src/docs/labels/cold-wallet/index.md [hunk 11]
```diff
@@ -7,14 +7,15 @@ description: Cold wallet label
...
 ## API Name
 
 `cold_wallet`
 
 ## Label Examples
-
+Bitfinex Cold Wallet Address: [0x742d35Cc6634C0532925a3b844Bc454e4438f44e](https://etherscan.io/address/0x742d35Cc6634C0532925a3b844Bc454e4438f44e)
```
[the remark this answers is not in the document]
Comment (vterentev) [reply src/docs/labels/cold-wallet/index.md:19]: You said no :)

**Pull request merged**